mesh-delivery/ios/AVPlayerMeshDelivery/README.md

```diff
@@ -1,19 +1,24 @@
 # Mesh Delivery SDK Integration for iOS, iPadOS and tvOS
 
 ## Prerequisite
+
```

Comment:
We should either not leave an empty line after headers everywhere, or leave them everywhere.  Idk how to see the pages without deploying to prod, so I'll let  you decide whichever is better :D

Reply:
I have a plugin in VSCode , which format the code once I press Ctrl+S/ Cmd+S.
For now I will "undo" this kind of format "just to be safe"